src/schema_tools/coerce.cljc

```diff
@@ -236,12 +235,17 @@
       (catch #?(:clj Exception, :cljs js/Error) _ x))
     x))
 
+(defn keyword->string [x]
+  (cond-> x
+          (keyword? x) name))
+
 (defn collection-matcher [schema]
   (if (or (and (coll? schema) (not (record? schema))))
     (fn [x] (if (coll? x) x [x]))))
 
 (def +json-coercions+
```

Comment:
could you add the keyword->x coercions also to json too? e.g. `{s/Int s/Int}` might come as `{:1 1}` over the wire. JSON can't represent keywords itself, so adding these keeps the coercion as strict as before, e.g. in the default case, just effects the keys.